docs/_includes/lightbox-dialog.html

```diff
@@ -2,12 +2,14 @@
     {% include section-header.html name="lightbox-dialog" version=page.versions.lightbox-dialog %}
 
     <p>A lightbox dialog is a modal window spawned by the main web page or application.</p>
+    <p>On small screens, the lightbox dialog will turn into a bottom sheet. The handle at the top will appear only on small screens.</p>
 
     <div class="demo">
         <div class="demo__inner">
             <button class="btn btn--primary dialog-button" data-makeup-for="lightbox-dialog-default" type="button">Open Lightbox</button>
             <div aria-labelledby="dialog-title-default" aria-modal="true" class="lightbox-dialog" hidden id="lightbox-dialog-default" role="dialog">
                 <div class="lightbox-dialog__window">
+                    <button class="lightbox-dialog__handle" type="button"></button>
```

Comment:
This button needs an aria-label. Maybe we go with aria-label="Resize Dialog"? Might be simpler than trying to toggle between "Maximize Dialog" & "Minimize Dialog", but those would also be fine if we prefer.

Reply:
Interestingly enough I copied this exactly from drawer dialog and the one place I copied it from did not have `aria-label`. I added it though.